README.md

```diff
@@ -521,22 +524,27 @@ Clients must use the fields from token response in subsequent calls to the [3. I
+| `alg` | (Algorithm) The cryptographic algorithm used to sign the JWT | y        | [RFC7515](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.1) | MUST be `EdDSA` or `ES256K`                                                 |
+| `kid` | (KeyID) The fully qualified DID Key ID of the signer         | y        | [RFC7515](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.4) | In the form `did:{method}:{identifier}#{key_id}`                            |
+| `typ` | (Type) The explicit JWT type                                 | y        | [RFC7515](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.9) | MUST be `at+jwt` per [RFC9068](https://www.rfc-editor.org/rfc/rfc9068.html) |
 
 ##### `access_token` Claims
 
 The `access_token` granted by the Credential Issuer contains the following [JWT Claim](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1) fields.
 
-| Field | Description                                                        | Required | References                                                             | Comments |
-| :---- | :----------------------------------------------------------------- | :------- | :--------------------------------------------------------------------- | :------- |
-| `iss` | (Issuer) The DID of the Credential Issuer                          | y        | [RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.1) |          |
-| `sub` | (Subject) The DID of the customer applying for KCC (Applicant DID) | y        | [RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.2) |          |
-| `exp` | (Expiration) The time at which the `access_token` expires          | y        | [RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4) |          |
-| `iat` | (IssuedAt) The time at which the `access_token` was granted        | y        | [RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.6) |          |
+| Field            | Description                                                          | Required | References                                                                                           | Comments |
+| :--------------- | :------------------------------------------------------------------- | :------- | :--------------------------------------------------------------------------------------------------- | :------- |
+| `iss`            | (Issuer) The DID of the Authorization Server                         | y        | [RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.1)                               |          |
+| `aud`            | (Audience) The Resource Identifier of the Credential Issuer          | y        | [RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3)                               |          |
```

Comment:
by resource identifier do you mean the issuer's DID?

Reply:
yep